documentation/modules/appendix/pages/best-practices.adoc

```diff
@@ -132,3 +132,44 @@ broken special characters.
 
 image::constraints-decision-tree.svg[width=100%]
 
+[[reuse-elements]]
+== Reuse Elements
+
+To ensure efficiency and maintainability, it is essential to reuse these elements across different Aspects
+*_WHENEVER_* possible.
```

Comment:
It is unclear what "these elements" are. Also, we should not stress "whenever" so much; it's more important to reuse elements "when it makes sense" (but we can't use this wording, since it's too generic). I propose a phrasing like this:

```suggestion
It is generally advisable to reuse definitions of existing model elements that describe the desired semantics. This not only ensures efficiency and maintainability, but more importantly, it explicitly expresses that the model builds upon a set of agreed-upon abstractions. Reuse of exising model element definitions makes it clear that your model talks about _the same thing_. On the other hand, introducing another definition of an allegedly already existing concept is equivalent to stating that the deliberately newly created definition does indeed mean something different. This is particularly useful when model elements are created for terminology that can have many different meanings in various contexts, such as the term "process".
```